src/colorizer/shaders/colorize_R32.frag

```diff
@@ -60,7 +60,7 @@ void main() {
 
   // Data buffer starts at 0, non-background segmentation IDs start at 1
   float featureVal = getFeatureVal(id - 1);
-  uint outlierVal = getOutlierVal(id - 1)
+  uint outlierVal = getOutlierVal(id - 1);
```

Comment:
??? how was this shader even compiling without the semicolon

Reply:
If this is a totally unused shader then maybe we should just delete the whole file

Reply:
Currently unused, yeah. I'll make that change.